functions/openhab.js

```diff
@@ -401,10 +480,12 @@ function syncAndDiscoverDevices(token, success, failure) {
 
 	// Checks for a Fahrenheit tag and sets the righ property on the
 	// attributeDetails response object
-	var setTempFormat = function(item, attributeDetails){
+	var setTempFormat = function(item, attributeDetails) {
 		if (item.tags.indexOf('Fahrenheit') > -1 || item.tags.indexOf('fahrenheit') > -1) {
```

Comment:
`item.tags.includes()`

Reply:
Perhaps this instead, two birds one code: item.tags.toString().toLowerCase().includes('fahrenheit')